src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -263,28 +248,12 @@ def _specific_row(self, slot, market):
         if type(self.area.strategy) == HeatPumpStrategy:
             return [
                 round(
-                    self.area.strategy.state.tanks.get_unmatched_demand_kWh(slot),
-                    ROUND_TOLERANCE,
-                ),
-                round(
-                    self.area.strategy.state.tanks.get_average_tank_temperature(slot),
+                    self.area.strategy.state.charger.tanks.get_average_tank_temperature(slot),
```

Comment:
I understand that making the `charger` member of the CombinedHeatPumpState class public was mandatory due to this, and for now it is fine. However, at some point it would be great if we used the output of `get_results_dict` here, instead of accessing the class directly. That way the charger member could stay private, and the data required here would still be accessible via the dict. IMO this applies to all strategy classes and not just for this one, but again probably it is better to be dealt in a separate story.

Reply:
Ah yes, I agree. Lets refactor this in a small separate story. 
I quickly created one: https://gridsingularity.atlassian.net/browse/GSYE-882

Reply:
Thanks!